anchor/programs/pubkey-protocol/src/utils.rs

```diff
@@ -10,7 +10,7 @@ pub fn is_valid_username(username: &str) -> bool {
 
     if !username
         .chars()
-        .all(|c| c.is_ascii_lowercase() || c.is_ascii_digit() || c == '_')
+        .all(|c| c.is_ascii_lowercase() || c.is_ascii_digit() || c == '_' || c == '-')
```

Comment:
Not too keen on adding more. I figured we use the same convention as X and Reddit. Having more options will lead to more confusion, easier to create a clone of a legit account.